skaffold.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: skaffold/v2beta6
+apiVersion: skaffold/v2beta29
```

Comment:
Bumped skaffold so we could use this convenient new flag as available in the latest version.